docs/MS Excel/Формат ячеек.md

```diff
@@ -0,0 +1,39 @@
+≡ Краткое руководство Markdown
+
+# Формат ячеек
+
+**Общий формат**
```

Comment:
[ch-md-error-emphasis-instead-of-a-heading](https://chaberch.github.io/how-to-chaberch/markdown/#ch-md-error-emphasis-instead-of-a-heading)